packages/@adobe/spectrum-css-temp/components/slider/index.css

```diff
@@ -69,7 +75,7 @@ governing permissions and limitations under the License.
   min-inline-size: var(--spectrum-slider-min-width);
 
   /* These calculations prevent the track from spilling outside of the control */
-  inline-size: calc(100% - calc(var(--spectrum-slider-controls-margin) * 2));
+  inline-size: calc(100% - calc(var(--spectrum-slider-controls-margin)));
```

Comment:
Since the default width is being specified on the wrapping div now, I _think_ you an revert this change here and remove the `margin-inline-end` that you added below

Reply:
danni/devon removed it